assembl/tests/views_tests/test_routes.py

```diff
@@ -251,6 +258,20 @@ def test_route_discussion_idea_legacy(discussion, root_post_1, subidea_1,
     assert resp.status_int == 200
 
 
+def test_route_discussion_post_v2(
+        test_app, discussion_with_2_phase_interface_v2,
+        post_related_to_sub_idea_1):
+    from urllib import quote_plus
+    slug = discussion_with_2_phase_interface_v2.slug
+    route = "/%s/posts/%s" % (
+        slug, quote_plus(post_related_to_sub_idea_1.uri()))
+    print route
+    resp = test_app.get(route)
+    assert resp.status_int == 303
+    headers = get_response_headers(resp)
+    assert ('/%s/debate/thread/theme/' % slug) in headers['Location']
```

Comment:
What about the share URL of a message that has been posted in another phase than the thread one? (for exemple multicolumns)